index.html

```diff
@@ -555,7 +556,102 @@ <h2>Dictionary {{RTCRtpEncodingParameters}} Members</h2>
         allows the <a>user agent</a> to adapt its bandwidth allocation strategy
         based on the current network conditions.</p>
       </dd>
+      <dt><dfn data-idl>codec</dfn> of type <span class="idlMemberType">RTCRtpCodec</span></dt>
+      <dd>
+        <p>Optional value overriding the codec used for this encoding. The {{RTCRtpCodec}} dictionary is
+          defined in [[WEBRTC]]. If absent, the default codec selection algorithm selects the first
+          negotiated codec.</p>
+          <div class="issue atrisk">
+            <p>
+              {{RTCRtpEncodingParameters/codec}} is marked as a feature at risk
+              due to lack of support from implementers.
+            </p>
+          </div>
+      </dd>
     </dl>
+    <h2>Modifications to existing procedures</h2>
+    <h3>setParameters()</h3>
+    <p>Add the following steps in the parameter validation for {{RTCRtpSender/setParameters()}},
+      before any validation of <var>scalabilityMode</var> from [[WEBRTC-SVC]]:</p>
+    <ol>
+      <li><p>Let <var>codecs</var> be <var> parameters</var>.{{RTCRtpParameters/codecs}}.</p>
+      </li>
+      <li>
+        <p>If <var>codecs</var> is not an empty list:</p>
+        <ol>
+          <li>
+            <p>If any <var>codec</var> parameter in <var>encodings</var> does not match
+              any entry in <var>codecs</var>, return a promise [= rejected =] with a newly
+              [= exception/created =] {{InvalidModificationError}}.</p>
+          </li>
+        </ol>
+      </li>
+      <li>
+        <p>Else (<var>codecs</var> is an empty list), if <var>transceiver</var>.
```

Comment:
Prose-wise we use "Otherwise," to be less codey — But I find the repeated "Else" pattern here confusing, (even more so with the exists and non-empty). Can we avoid it?